src/index.js

```diff
@@ -93,6 +95,8 @@ class IOpipeWrapperClass {
       }
     });
 
+    invocationContext = this.context;
```

Comment:
Two examples. I believe we can fix the first example easily. I do not see a simple solution to the second.

The first may be solved by storing any pre-defined values for invocationContext and resetting upon exiting the function/invocation. This would also provide safety against issues across two invocations (across time).

Example 1:
```
const iopipeModule =require('@iopipe/iopipe')
const iopipe = iopipeModule()
const f1 = iopipe((event, context) => {
  f2()
  iopipe.getContext() // would return context for f2()
})

const f2 = iopipe((event, context) => { })
```

Example 2:
```
const iopipeModule =require('@iopipe/iopipe')
const iopipe = iopipeModule()
const f1 = iopipe((event, context, callback) => {
  f2(event, context, () => {
    iopipe.getContext() // would return context for f2() 
    callback()
  })
})

const f2 = iopipe((event, context, callback) => { callback() })
```